nautobot/docs/release-notes/version-2.0.md

```diff
@@ -10,6 +10,12 @@ If you are a user migrating from Nautobot v1.X, please refer to the ["Upgrading
 
 ### Added
 
+#### Collapse Region and Site Models into Location ([#2517](https://github.com/nautobot/nautobot/issues/2517))
+
+##### Move Site Fields to Location ([#2954](https://github.com/nautobot/nautobot/issues/2954))
```

Comment:
```suggestion
#### Added Site Fields to Location ([#2954](https://github.com/nautobot/nautobot/issues/2954))
```

#2517 will probably be a `Changed` or `Removed` item rather than `Added`.